.github/workflows/binaries.yml

```diff
@@ -0,0 +1,91 @@
+name: Binaries
+
+on:
+  pull_request:
+
+jobs:
+  binaries:
+    strategy:
+      matrix:
+        os:
+          - ubuntu
+          - macOS
+          # - windows
+
+        ghc:
+          - "9.2"
+          - "9.0"
+          - "8.10"
```

Comment:
Do the `hie` files fail only on major version bumps? They won't fail on a difference between `9.2.2` and `9.2.4`?

Reply:
Good question. I thought so, but I will confirm.

Reply:
Fuuuuuuuck

```
incompatible hie file: /home/patrick/code/freckle/megarepo/tts/.stack-work/dist/x86_64-linux/Cabal-3.6.3.0/build/tts/tts-tmp/Paths_tts.hie
    this version of weeder was compiled with GHC version 9002
    the hie files in this project were generated with GHC version 9025
    weeder must be built with the same GHC version as the project it is used on
```

Even though this is a 9.0/9.2 difference, the fact that it shows full version numbers and says "the same GHC version", make me think it has to match exactly.

It seems like I'll have to build binaries for each specific GHC version to accomplish my goal here. I'm still leaning towards doing it, but this PR is not worth merging as is.

Reply:
I'm in favor of building the binaries too :+1: